VERSION

```diff
@@ -1 +1 @@
-0.3.1
+0.3.2
```

Comment:
I would wait for actually releasing a new version before updating this, at the very least with the goal of making sure we do not end up bumping version number multiple times before release.

Reply:
I was not sure either, but I interpreted the third digit of the version as patching without introducing new features (see https://semver.org/). It was meant to happen after this commit + codestyle where more noticeable changes would happen.
I'm fine with not increasing to 0.3.2 in this MR.